migrations/Version202109281632856186_taoLti.php

```diff
@@ -0,0 +1,51 @@
+<?php
```

Comment:
Should we also create an install script for this?

Reply:
as I know, we should not